utils/extraction_utils.py

```diff
@@ -0,0 +1,155 @@
+"""
+This file contains functions to add single cell count metadata into the returned files or before 
+saving the dataframe. As well, there are functions to extract image features from the outputted 
+sqlite file from CellProfiler (based on the functions from the cells.SingleCells class in Pycytominer).
+"""
+
+# must use the annotations import as CellProfiler is restricted to Python 3.8 at this time so Optional
+# by itself only works in Python 3.10
+from __future__ import annotations
+from typing import Optional
+import pathlib
+import pandas as pd
+from sqlalchemy import create_engine
+import numpy as np
+
+from pycytominer import normalize, feature_select
+from pycytominer.cyto_utils import cells, output
```

Comment:
Consider removing the imports which aren't used here.

Reply:
👍